src/helper/one/profiler/JfrSync.java

```diff
@@ -72,40 +88,43 @@ public static void stop() {
     }
 
     private static void disableBuiltinEvents(Recording recording, int eventMask) {
-        if ((eventMask & 1) != 0) {
+        if ((eventMask & EM_CPU) != 0) {
             recording.disable("jdk.ExecutionSample");
             recording.disable("jdk.NativeMethodSample");
         }
-        if ((eventMask & 2) != 0) {
+        if ((eventMask & EM_ALLOC) != 0) {
             recording.disable("jdk.ObjectAllocationInNewTLAB");
             recording.disable("jdk.ObjectAllocationOutsideTLAB");
             recording.disable("jdk.ObjectAllocationSample");
             recording.disable("jdk.OldObjectSample");
         }
-        if ((eventMask & 4) != 0) {
+        if ((eventMask & EM_LOCK) != 0) {
             recording.disable("jdk.JavaMonitorEnter");
             recording.disable("jdk.ThreadPark");
         }
-        if ((eventMask & 8) != 0) {
+        // No built-in event related to EM_WALL
+        // No built-in event related to EM_NATIVEMEM
+        if ((eventMask & EM_METHOD_TRACE) != 0) {
             recording.disable("jdk.MethodTrace");
         }
 
+        eventMask >>= (EVENT_MASK_SIZE - 1);
```

Comment:
`>>>=` and no `-1`

Reply:
ff8b4447a684a3d915c3eb2243425589c295c7b1